src/share.js

```diff
@@ -67,17 +70,28 @@ class Share extends BasePlugin {
   }
 
   _addIcon() {
-    this.player.ready().then(() => {
-      const ShareWrapper = () => <ShareButton config={this.config} setRef={this._setPluginButtonRef.bind(this)} />;
-      this.iconId = this.player.getService('upperBarManager').add({
-        displayName: 'Share',
-        ariaLabel: <Text id="controls.share">Share</Text>,
-        order: 70,
-        component: ShareWrapper,
-        svgIcon: {path: ICON_PATH},
-        onClick: this._openShareOverlay.bind(this)
+    // @ts-ignore
+    if (ui.redux.useStore().getState().shell['activePresetName'] !== ReservedPresetNames.MiniAudioUI) {
```

Comment:
this one uses across all plugins, I feel like we can add utility `isMiniAudioUiPreset` that can be exported and re-used in plugins

Reply:
good idea, for the next release/session, unfortunately in this release the focus is pure functional